src/js/_store/modules/categories.js

```diff
@@ -5,17 +5,32 @@ export default {
   state: {
     categoryList: [],
     errorMessage: '',
+    doneMessage: '',
+    loading: false,
   },
   mutations: {
+    clearMessage(state) {
+      state.errorMessage = '';
+      state.doneMessage = '';
+    },
+    toggleLoading(state) {
+      state.loading = !state.loading;
+    },
     doneGetAllCategories(state, payload) {
       state.categoryList = [...payload.categories];
       state.categoryList = state.categoryList.reverse();
     },
+    createCategory(state) {
+      state.doneMessage = '新規カテゴリーを作成しました。';
+    },
```

Comment:
[imo]

中の処理でやってることとしては、「新規作成完了後のメッセージを doneMessage にセットする」ということだと思うので、メソッド名は以下が良いかと思います。

```suggestion
    setCreatedDoneMessage(state) {
      state.doneMessage = '新規カテゴリーを作成しました。';
    },
```